pkg/cloudprovider/providers/azure/azure_instances.go

```diff
@@ -23,10 +23,9 @@ import (
 	"strings"
 
 	"k8s.io/api/core/v1"
-	"k8s.io/kubernetes/pkg/cloudprovider"
-
-	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/types"
+	cloudprovider "k8s.io/cloud-provider"
```

Comment:
could you change this back to `"k8s.io/kubernetes/pkg/cloudprovider"`

Reply:
Yeah, I'm going to close this PR out and attempt to do a cherry pick in a new PR. My IDE did some opinionated changes

Reply:
I fixed it on this pull request